### PR TITLE
Issue 68

### PR DESCRIPTION
**Pull request by onmax**
#68

### CODE REVIEW

File: congress/static/congress/angular/app.js [hunk 1]
```diff
@@ -211,14 +211,38 @@
 		// Boolean used for see if data have loaded
 		$scope.hasData = false
 
+		$scope.currentYears = []
+		let currentYear = new Date().getFullYear()
+		const counterYear = 2016
+		while(counterYear <= currentYear)
```
Comment (alvarogtx300) [inline congress/static/congress/angular/app.js]: This block without braces can be a bit confusing

File: congress/static/congress/angular/app.js [hunk 2]
```diff
@@ -211,14 +211,38 @@
...
+			$scope.currentYears.push(currentYear--)
+		
 
 		$scope.searchECTS = function (){
 			$scope.dni_nie_error = validateNIF_NIE($scope.dni_nie)
+			
 			if($scope.dni_nie_error === ""){
-				fetch(`${window.location.href}${$scope.dni_nie}`)
+				let url = window.location.href
+				if (url.substring(url.length - 1) === '?')
```
Comment (alvarogtx300) [inline congress/static/congress/angular/app.js]: Same as above

File: congress/static/congress/angular/app.js (hunk 1 above)
Comment (onmax) [reply congress/static/congress/angular/app.js]: But without braces means that only execute one line. I think is not confusing at all

Comment (alvarogtx300) [reply congress/static/congress/angular/app.js]: It's just a matter-of-taste, I'm used to writing the braces for every block.
But feel free to leave it as it is.

Comment (onmax) [reply congress/static/congress/angular/app.js]: Done

**Pull request merged**